extensions/wikia/PortableInfobox/services/Parser/Nodes/NodeTitle.php

```diff
@@ -4,9 +4,24 @@
 class NodeTitle extends Node {
 	public function getData() {
 		if ( !isset( $this->data ) ) {
-			$this->data = [ 'value' => $this->getValueWithDefault( $this->xmlNode ) ];
+			$title = $this->sanitizeInfoboxTitle( $this->getValueWithDefault( $this->xmlNode ) );
```

Comment:
We want to strip HTML markup from title only on mobile ( especially mercury ). This change will do it everywhere. Please limit the sanitizing to `wikiamobile` skin only